client/src/components/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import axios from 'axios';
+
+import './LoginForm.css';
+
+const LoginForm = ({setToken}) => {
+    const {
+        register,
+        handleSubmit,
+        formState: { errors }
+    } = useForm();
+
+    const [success, setSuccess] = useState("");
+    const [error, setError] = useState("");
+
+
+    const LogUser = async (email, password) => {
+        const user = {
+            password: password,
+            email: email
+        }
+        const api = axios.create({
+            baseUrl: 'http://localhost:7000'
+        })
+        await api.post('/api/auth/login', user)
+            .then(response => {
+                let data = response.data
+                if (data.success) {
+                    setSuccess(data.isAuth)
+                    return window.location = "/";
+                }
+                if (data.error) return setError(data.message)
+                console.log(data)
+
+            })
+            .catch(err => {
+                setError(err.data.message)
+            })
+    }
+    return (
+        <form action="" onSubmit={handleSubmit((data) => {
+            LogUser(data.email, data.Password)
+        })}>
+            <div>
+                <label className={errors.email?.message || error || success}>Email</label>
+                <small className={errors.email?.message || error}>{errors.email?.message || error}</small>
+                <input type="email" {...register("email", {
+                    required: "email is required",
+                    pattern: {
+                        value: /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i,
+                        message: "invalid email address"
+                    }
+                }
+                )}></input>
+            </div>
```

Comment:
This is very similar code to the password as well, why no to turn it into a component and reuse it?